### PR TITLE
Feat: Add Cerebellum

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

**Comment by arnaudon** [comment]
This is based on #18

File: atlas_placement_hints/app/placement_hints.py [hunk 1]
```diff
@@ -340,3 +340,59 @@ def thalamus(
...
+@app.command()
+@verbose_option
+@common_atlas_options
+@click.option(
+    "--metadata-path",
+    type=EXISTING_FILE_PATH,
+    required=False,
+    help=(
+        "(Optional) Path to the metadata json file. Defaults to "
+        f"`{str(METADATA_REL_PATH / 'cerebellum_metadata.json')}`"
```
Comment (mgeplf) [inline atlas_placement_hints/app/placement_hints.py:354]: is the `str` here needed?

Comment (arnaudon) [reply atlas_placement_hints/app/placement_hints.py:354]: ah no, I just copied another entry, I can remove them yep

**Comment by arnaudon** [comment]
I'll rebase!

**Comment by arnaudon** [comment]
voila!

**Review by mgeplf — APPROVED** [review]
can't really comment on the correctness, you're the scientist :)

**Comment by arnaudon** [comment]
yes, I did my scientists, copy pasting previous code, and randomly changing things until it does something slightly different :)

**Pull request merged**